src/applications/disability-benefits/disability-rating-calculator/tests/00-all-fields.e2e.spec.js

```diff
@@ -29,8 +28,6 @@ module.exports = E2eHelpers.createE2eTest(client => {
     .waitForElementPresent(componentSelector, Timeouts.slow);
 
   // Test row deletion.
-  // TODO: Once Design Team has spec'ed out the relevant UX,
-  // cover edge-case for deleting row AFTER calculating combined-rating.
```

Comment:
Deleting this TODO comment - UX has been alerted to this edge-case, but has not committed to addressing it.

Reply:
Looks like we just need to change the `drcPagePath` value to be `'/disability/about-disability-ratings/` and I think we're good to go!

Reply:
Just fixed that & pushed it up.  :)